pallets/handles/src/lib.rs

```diff
@@ -178,14 +180,14 @@ pub mod pallet {
 	#[pallet::event]
 	#[pallet::generate_deposit(pub (super) fn deposit_event)]
 	pub enum Event<T: Config> {
-		/// Deposited when a handle is claimed. [MSA id, full handle in UTF-8 bytes]
+		/// Deposited when a handle is claimed. [MSA id, display handle in UTF-8 bytes]
 		HandleClaimed {
 			/// MSA id of handle owner
 			msa_id: MessageSourceId,
 			/// UTF-8 string in bytes
```

Comment:
Doc suggestion to be more clear: (Also for the HandleRetired)
```suggestion
			/// UTF-8 string in bytes of the display handle
```